torch/csrc/jit/runtime/register_ops_utils.cpp

```diff
@@ -369,6 +370,31 @@ void listAdd(Stack* stack) {
   push(stack, std::move(ret));
 }
 
+void tupleAdd(Stack* stack) {
+  auto b = pop(stack).toTuple();
+  auto a = pop(stack).toTuple();
+
+  std::vector<IValue> result;
+
+  for (const auto& el : a->elements()) {
+    // don't add None to tuple
+    if (!el.isNone()) {
```

Comment:
Why skipping `None`s? I tried following python code, which seems to not skip `None`
```
>>> a = (1, None, 2)
>>> b = (3, None, 4)
>>> a + b
(1, None, 2, 3, None, 4)
```